### ISSUE TITLE
Accessibility telemetry

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#10179]
As a product manager, I want to understand how often accessibility features are used.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-3874)

**Comment by data-sync-user** [comment]
➤ Nishant Bhasin commented:

Adding tracking for events when the app opens is a good place. 

Ideally whatever View Controller gets initialized first (Ex BrowserViewController) should have it inside their view did load.

Ex:

func trackAccessibility() {
    print("UIAccessibility.isVoiceOverRunning - \(UIAccessibility.isVoiceOverRunning)")
    print("UIAccessibility.isReduceTransparencyEnabled - \(UIAccessibility.isReduceTransparencyEnabled)")
    print("UIAccessibility.isReduceMotionEnabled - \(UIAccessibility.isReduceMotionEnabled)")
    print("UIAccessibility.isInvertColorEnabled - \(UIAccessibility.isInvertColorsEnabled)")
    print("UIAccessibility.isSwitchControlRunning - \(UIAccessibility.isSwitchControlRunning)")
}

**Comment by data-sync-user** [comment]
➤ Laurie Marceau commented:

Not sure why it doesn’t appear in JIRA, but the linked github ticket is https://github.com/mozilla-mobile/firefox-ios/issues/10179 ( https://github.com/mozilla-mobile/firefox-ios/issues/10179|smart-link )

**Comment by data-sync-user** [comment]
➤ Simion Basca commented:

We were unable to view the mentioned flags in the metrics ping in Glean. I my case after enabling voice over or any other accessibility feature from the acceptance criteria, I am unable to see them in the metrics ping. 

Sorin Paraipan could you please double check if this is working correctly?

**Comment by data-sync-user** [comment]
➤ Sorin Paraipan commented:

Hi Simion Basca , sure I’ll take a look to see what is happening.

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Verified as fixed on the latest 9000 build (10213) with iPhone 13 Pro Max (15.1.1).

!image-20220609-122052.png|width=637,height=892!

!image-20220609-122056.png|width=566,height=544!

I was able to generate all the mentioned events in the spreadsheet.
